### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/cookie-params.md`

### PR DESCRIPTION
**Pull request by hard-coders**
This PR translates `tutorial/cookie-params.md` in Korean.

related: #2017

### CODE REVIEW

**Review by PandaHun — APPROVED** [review]
It's perfect done

File: docs/ko/docs/tutorial/cookie-params.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+이제 `Path` 및 `Query`를 사용한 동일한 구조를 이용하여 쿠키 매개변수를 선언합니다.
+
+첫 번째 값은 기본값이며, 추가 검증이나 어노테이션 매개변수 모두 전달할 수 있습니다:
+
+```Python hl_lines="9"
+{!../../../docs_src/cookie_params/tutorial001.py!}
+```
+
+!!! note "기술 세부사항"
```
Comment (0417taehyun) [inline docs/ko/docs/tutorial/cookie-params.md:23]:
```suggestion
!!! note "기술적 세부사항"
```

Check [파이썬 공식 한국어 문서에 없는 용어 중 현재 사용하고 있는 용어
](https://github.com/tiangolo/fastapi/discussions/3167#discussioncomment-721448)

**Review by 0417taehyun — CHANGES_REQUESTED** [review]
Bit appreciate of your works 🚀 

For solving some conflicts, can you update this file to latest version?

Feel free to check my reviews, it is just a SIMPLE thing.

**Comment by tiangolo** [comment]
📝 Docs preview for commit d049d590e1033374f54d0f2b62824f0f086a16be at: https://6499e2be5f77290548b8a60d--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**